app/src/modules/settings/routes/roles/permissions-detail/components/presets.vue

```diff
@@ -47,7 +58,32 @@ export default defineComponent({
 			},
 		});
 
-		return { t, presets };
+		/**
+		 * Interfaces that use the useRelationMultiple composable.
+		 */
+		const relationalInterfaces = ['list-m2a', 'list-m2m', 'list-o2m', 'list-o2m-tree-view', 'files'];
```

Comment:
We can't hardcode interfaces by name, as interfaces are an extension type. The only way to resolve that is by looking up the details of each registered interface to extract a list of relational interfaces